nym-vpn-core/crates/nym-vpnd/src/service/vpn_service.rs

```diff
@@ -35,13 +35,13 @@ use nym_vpn_lib::{
 use nym_vpn_lib_types::{TunnelEvent, TunnelState, TunnelType};
 use zeroize::Zeroizing;
 
-use crate::config::GlobalConfigFile;
-
 use super::{
     config::{NetworkEnvironments, NymVpnServiceConfig, DEFAULT_CONFIG_FILE},
     error::{AccountError, Error, Result, SetNetworkError, VpnServiceDeleteLogFileError},
     VpnServiceConnectError, VpnServiceDisconnectError,
 };
+use crate::config::GlobalConfigFile;
```

Comment:
Is this reordering done by the formatter? I think they were ordered like that for a reason

Reply:
I run some cargo fmt